SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialSender.h

```diff
@@ -119,8 +122,11 @@ class SPATIALGDK_API USpatialSender : public UObject
 
 	void FlushPackedUnreliableRPCs();
 
+	void GainAuthorityThenAddComponent(USpatialActorChannel* Channel, UObject* Object, const FClassInfo* Info);
```

Comment:
This is an odd function name. Does it mean wait till authority is gained then add the component. I'd prefer it if this is was no baked into the sender.